tests/drivers/gpio/gpio_ite_it8xxx2_v2/src/main.c

```diff
@@ -0,0 +1,462 @@
+/*
+ * Copyright 2023 The ChromiumOS Authors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/device.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/dt-bindings/gpio/ite-it8xxx2-gpio.h>
+#include <zephyr/fff.h>
+#include <zephyr/ztest.h>
+
+#define MY_SERIAL DT_NODELABEL(gpioa)
+
+const struct device *const gpio_dev = DEVICE_DT_GET(MY_SERIAL);
+
+/* These values must match what is set in the dts overlay. */
+#define TEST_PIN  1
+#define TEST_IRQ  2
+#define TEST_MASK BIT(4)
```

Comment:
TEST_PIN is arbitrary, but TEST_IRQ and TEST_MASK can be read from the device tree.
```
#define TEST_IRQ DT_IRQ_BY_IDX(DT_NODELABEL(gpioa), TEST_PIN, irq)
```

Reply:
done